lib/arc.js

```diff
@@ -24,11 +24,11 @@ export function configure(options) {
   options; // for lint
   // not used at the moment:  const network = options && options.network && options.network.name ? options.network.name : 'testrpc';
   /**
-     * TODO: supply testrpc url in options?  Problem is that at this point web3 has already been set in utils
-     * so it's too late at this point to set the url.  Would need to somehow effect this initialization before
-     * any contract imports have been attempted.  Need to figure out how to export the modules above somewhere
-     * else, or separately.
-     */
+   * TODO: supply testrpc url in options?  Problem is that at this point web3 has already been set in utils
```

Comment:
why need testrpc url ?

Reply:
This PR is about formatting, that is all

Reply:
Because of this code in utils.js:

```
    preWeb3 = new Web3(
      new Web3.providers.HttpProvider("http://localhost:8545")
    );
```